### PR TITLE
[WIP] add node & rollbar examples, etc.

### PR DESCRIPTION
**Pull request by iynere**
documenting the last few orbs that are missing examples...

### CODE REVIEW

File: src/node/orb.yml [hunk 1]
```diff
@@ -3,6 +3,27 @@ version: 2.1
 description: |
```
Comment (eddiewebb) [inline src/node/orb.yml]: I thought our node orb came from eugene and has jobs and other stuff.

Comment (iynere) [reply src/node/orb.yml]: this is the one from eugene @eddiewebb

Comment (iynere) [reply src/node/orb.yml]: unless whatever was in a separate repo had more—but i thought that got merged into the monorepo

**Comment by felicianotech** [comment]
@iynere Would you like to rebase this PR? Or is this dead now?

**Comment by iynere** [comment]
good question @felicianotech 

let me take a look at it again once i finish w/ binary authorization orb

**Pull request closed without merging**